test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1

```diff
@@ -24,139 +24,125 @@ Describe "XmlCommand DRT basic functionality Tests" -Tags "CI" {
             public string Property2;
         }
 "@
-		}
+        }
     }
 
-	BeforeEach {
-		$testfile = Join-Path -Path $TestDrive -ChildPath "clixml-directive.xml"
-	}
+    BeforeEach {
+        $testfile = Join-Path -Path $TestDrive -ChildPath "clixml-directive.xml"
+    }
 
     AfterEach {
-		remove-item $testfile -Force -ErrorAction SilentlyContinue
+        remove-item $testfile -Force -ErrorAction SilentlyContinue
     }
 
- 	It "Import with CliXml directive should work" {
+     It "Import with CliXml directive should work" {
         Get-Command export* -Type Cmdlet | Select-Object -First 3 | Export-Clixml -Path $testfile
-		$results = Import-Clixml $testfile
-		$results.Count | Should -BeExactly 3
+        $results = Import-Clixml $testfile
+        $results.Count | Should -BeExactly 3
         $results[0].PSTypeNames[0] | Should -Be "Deserialized.System.Management.Automation.CmdletInfo"
     }
 
-	It "Import with Rehydration should work" {
-		$property1 = 256
-		$property2 = "abcdef"
-		$isHiddenTestType = [IsHiddenTestType]::New($property1,$property2)
-		$isHiddenTestType | Export-Clixml $testfile
-		$results = Import-Clixml $testfile
-		$results.Property1 | Should -Be $property1
-		$results.Property2 | Should -Be $property2
+    It "Import with Rehydration should work" {
+        $property1 = 256
+        $property2 = "abcdef"
+        $isHiddenTestType = [IsHiddenTestType]::New($property1,$property2)
+        $isHiddenTestType | Export-Clixml $testfile
+        $results = Import-Clixml $testfile
+        $results.Property1 | Should -Be $property1
+        $results.Property2 | Should -Be $property2
+    }
+
+    It "Export-Clixml StopProcessing should succeed" {
+        $sb = [scriptblock]::Create("1..10000 | Export-CliXml -Path $testfile -Verbose")
```

Comment:
Isn't it better to use:

```powershell
$sb = { 1..10000 | Export-CliXml -Path $testfile -Verbose }
```

This will give syntax coloring and can be analyzed by psscriptanalyzer in vscode

Reply:
Done.